src/compressed_tensors/quantization/lifecycle/calibration.py

```diff
@@ -53,7 +53,19 @@ def set_module_for_calibration(module: Module, quantize_weights_upfront: bool =
 
     if quantize_weights_upfront and module.quantization_scheme.weights is not None:
         # set weight scale and zero_point up front, calibration data doesn't affect it
+        if not hasattr(module, "weight_observer"):
+            from compressed_tensors.quantization.lifecycle.initialize import (
```

Comment:
Are there significant import/dependency issues we incur when importing the initialize? Otherwise, it'd be great to keep this as a high-level import to make things a bit more readable and setup better for potential edits and refactors

Reply:
This is to prevent circular imports.